bazel/gflags.bzl

```diff
@@ -64,19 +70,28 @@ def gflags_library(hdrs=[], srcs=[], threads=1):
         "-DHAVE_SYS_TYPES_H",
         "-DHAVE_INTTYPES_H",
         "-DHAVE_SYS_STAT_H",
-        "-DHAVE_UNISTD_H",
-        "-DHAVE_FNMATCH_H",
         "-DHAVE_STRTOLL",
         "-DHAVE_STRTOQ",
-        "-DHAVE_PTHREAD",
         "-DHAVE_RWLOCK",
-    ]
+    ] + select({
+        "//:x64_windows": [
+            "-DOS_WINDOWS",
+        ],
+        "//conditions:default": [
+            "-DHAVE_UNISTD_H",
+            "-DHAVE_FNMATCH_H",
+            "-DHAVE_PTHREAD",
+        ],
+    })
     linkopts = []
     if threads:
-        linkopts.append("-lpthread")
+        linkopts = linkopts + select({
```

Comment:
Just out of curiosity and because you also changed `copts.append` later on to use `+` operator for lists instead: Is it possible to write this more concise (and possibly readable) as `linkopts += select(...)` (and `copts += ["-DNO_THREADS"]` below? Nevermind if not.

Reply:
Looks like that works fine.